src/amo/components/ServerHtml/index.js

```diff
@@ -72,6 +71,17 @@ export default class ServerHtml extends Component {
           );
         case 'js':
           return <script key={type + index} src={filePath} {...sriProps} />;
+        case 'font':
+          return (
+            <link
+              href={filePath}
+              {...sriProps}
+              key={type + index}
+              rel="preload"
+              as="font"
+              type="font/woff2"
```

Comment:
Is this going to be correct all the time?

Reply:
It should. We only want this strategy for woff2 fonts (I'm not interested in super old browsers which would likely not deal with `unicode-range` correctly and end up downloading all fonts all the time)